src/types/transaction_id.ts

```diff
@@ -0,0 +1,23 @@
+const trxIdRegex = /^([a-zA-Z]|[0-9]|-|_){43}$/;
+
+export class TransactionID {
+	constructor(private readonly transactionId: string) {
+		if (!transactionId.match(trxIdRegex)) {
+			throw new Error(
+				'Transaction ID should be a 43-character, alphanumeric string potentially including "=" and "_" characters.'
```

Comment:
> potentially including "="

Because it's a base64. Where do we trim the trailing equals?

Reply:
Should we accept it in the regex?

Reply:
I am following up on what number domain trx IDs are in.

Reply:
Based on convos with Arweave team in Discord it seems that =, though a valid Base64 URL character, would only appear at the end of a hashed string, and only if padding were required. Apparently no padding is required for the hashes we create here.

Reply:
We are covered here. The network doesn't require the trailing equals